website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -178,6 +178,8 @@ The following arguments are supported:
 
 * `service_principal` - (Required) A Service Principal block as documented below.
 
+* `enable_rbac` - (Optional) True or False. Enables or Disables Kubernetes Role Based Access Control (RBAC). Defaults to True. Changing this forces a new resource to be created.
```

Comment:
`Enables or disables`. If defaults to true but `aad_profile` is not set? What's the expected behavior?

Reply:
Updated to be more clear.